deployment-examples/aws/k8s/README.md

```diff
@@ -9,44 +9,45 @@ The main goal of this deploy process is to build and publish the new docker imag
-| `AWS_ECR_PATH`       | AWS ECR repository url(without repository name).                            |
-| `AWS_CLUSTER_NAME`   | AWS Cluster name.                                                           |
-| `CTP_PROJECT_KEY`    | Commercetools project key.                                                  |
-| `CTP_CLIENT_ID`      | Client Id.                                                                  |
-| `CTP_CLIENT_SECRET`  | Client secret.                                                              |
+| Name                              | Description                                                                 |
+| --------------------------------- | --------------------------------------------------------------------------- |
+| `AWS_ECR_REPOSITORY`              | Amazon Elastic Container Registry(ECR) name(It is also used as Image name). |
+| `AWS_REGION_CODE`                 | Same region code where cluster is created.                                  |
+| `AWS_ECR_PATH`                    | AWS ECR repository url(without repository name).                            |
+| `AWS_CLUSTER_NAME`                | AWS Cluster name.                                                           |
+| `CTP_PROJECT_KEY`                 | Commercetools project key.                                                  |
+| `CTP_CLIENT_ID`                   | Client Id.                                                                  |
+| `CTP_CLIENT_SECRET`               | Client secret.                                                              |
+| `SUBSCRIPTION_ORDER_CREATION_URL` | Subscription order creation url.                                            |
+| `CUSTOM_HEADERS`                  | Custom headers passed for subscription order creation.                      |
 
 ### Helm values
 
 For any changes for cronjob, configure the properties in [values.yaml](./values.yaml).
 
-## Create release.
+## Creating AWS cluster (only for the first time)
+
+Configure all the required environment variables in the bash script [`create_aws_cluster`](create_aws_cluster.sh). After that, execute the script file.
 
-To release the library, you need to ["create a new release"](https://github.com/commercetools/commercetools-subscriptions/releases/new) with Github,
-describe the new release and publish it.
+```bash
+./create_aws_cluster.sh
+```
 
-Bash scripts have been created to build and deploy docker image to AWS EKS as a cronjob. CD github actions job will run these bash scripts to deploy the new version to AWS.
+## Deploy to AWS cluster
 
-> In the deployment script, the [cron-job](https://github.com/commercetools/k8s-charts/tree/master/charts/cronjob) helm chart is used.
+Configure all the required environment variables in the bash script [`deploy_to_eks.sh`](deploy_to_eks.sh) and in the helm [values](values.yaml) file. After that, execute the script file.
```

Comment:
Why do you need to configure twice, once in the bash script and once in the helm file? Can't you do it once only, either helm file or the bash script file?

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/3e991b3f764a0379429497d4d64e9972c79f7807